src/ai/backend/manager/cli/image_impl.py

```diff
@@ -192,3 +194,58 @@ async def dealias(cli_ctx, alias):
             log.exception("Alias not found.")
             return
         await session.delete(alias_row)
+
+
+async def validate_image_alias(cli_ctx, alias: str) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
+    ):
+        try:
+            image_row = await ImageRow.from_alias(session, alias)
+            if image_row is None:
+                raise UnknownImageReference
+
+            for key, value in validate_image_labels(image_row.labels).items():
+                if key[:11] == "ai.backend.":
+                    print(f"{key[11:]:<30}: ", end="")
+                else:
+                    print(f"{key:<30}: ", end="")
+                print(*value) if isinstance(value, list) else print(value)
+
+        except UnknownImageReference:
+            log.error(f"No images were found with alias: {alias}")
+        except Exception:
+            log.exception("An error occurred.")
+
+
+async def validate_image_canonical(
+    cli_ctx, canonical: str, architecture: Optional[str] = None
+) -> None:
+    async with (
+        connect_database(cli_ctx.local_config) as db,
+        db.begin_session() as session,
```

Comment:
```suggestion
        db.begin_readonly_session() as session,
```
Same here, we only do `SELECT` job from the DB session